fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneIndexTest.java

```diff
@@ -2726,7 +2733,7 @@ private static void assertOnlyCompoundFileExisting(Index index, @Nonnull Subspac
                     || file.endsWith(".pky")) {
                 assertFalse(fileReference.getContent().isEmpty(), "fileName=" + file);
             } else {
-                assertTrue(FDBDirectory.isCompoundFile(file) || file.startsWith(IndexFileNames.SEGMENTS),
+                assertTrue(FDBDirectory.isCompoundFile(file) || file.startsWith(IndexFileNames.SEGMENTS) || FDBDirectory.isStoredFieldsFile(file),
```

Comment:
Should this be around? Shouldn't the `fsf` be deleted when it is added to the `cfs`?

Reply:
The `fsf` can be found here from another (e.g. future) segment. The `fsf` is created early enough in the process such that the next segments' one is already here, so added the segment name to the method to ensure that the `fsf` from the verified segment is not around